pkg/asset/machines/vsphere/machines.go

```diff
@@ -40,8 +41,28 @@ func Machines(clusterID string, config *types.InstallConfig, pool *types.Machine
 		replicas = *pool.Replicas
 	}
 
+	// Create hosts to populate from.  Copying so we can remove without changing original
+	// and only put in the ones that match the role.
+	var hosts []*vsphere.Host
+	if config.Platform.VSphere.Hosts != nil {
+		for _, host := range config.Platform.VSphere.Hosts {
+			logrus.Debugf("host.role=%v role=%v", host.Role, role)
```

Comment:
Maybe we want to drop this debug statement? This looks less polished than the others, which seem good.

Reply:
yep, good catch